### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by sanket-bhandari**
Sanket's contribution

### CODE REVIEW

File: Misleading_Visualizations_Survey.Rmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Is the data visualization misleading or not - Survey 
+
+Conducted by:
```
Comment (jl6298) [inline Misleading_Visualizations_Survey.Rmd:3]: Please write your names in one line. （third line）

**Review by jl6298 — CHANGES_REQUESTED** [review]

File: Misleading_Visualizations_Survey.Rmd (hunk 1 above)
Comment (jl6298) [reply Misleading_Visualizations_Survey.Rmd:3]: No worry, you are not passing the submission date.

**Comment by jtr13** [comment]
@sanket-bhandari Please make the changes as requested so the PR can be merged.

**Comment by sanket-bhandari** [comment]
Professor, I have created a new pull request (https://github.com/jtr13/cc22tt/pull/127) and made changes to that one.
Sorry for the trouble.

**Comment by jtr13** [comment]
Thank you for your contribution! 🎉

**Pull request merged**